docs/Advanced-Usage/Workloads/Defining-Custom-Workloads.rst

```diff
@@ -1,208 +0,0 @@
-.. _defining-custom-workloads:
```

Comment:
can you make this point to whatever the new relevant documentation is? there are outside links to these pages